src/space/objects/realization/blackhole.js

```diff
@@ -0,0 +1,91 @@
+import Vector from "../../utils/vector";
+import EncounterObject from "../pattern/encounterObject";
+import Type from "../pattern/encounterObjectType";
+import MyMath from "../../utils/myMath";
+import AssetLoader from "../../utils/assetLoader";
+
+const BH_SPRITE_RADIUS = 50;
+const BH_FORCE_SCALE = 600.000 / 1000.000;
```

Comment:
Why not use just `6.0 / 10.0;`?

Reply:
or just 0.6, since you are using decimal numbers anyway